.cargo/config

```diff
@@ -15,4 +15,7 @@ rustflags = [
     "-Dclippy::todo",
     "-Dclippy::unnecessary_wraps",
     "-Dclippy::useless_transmute",
+    "-Delided_lifetimes_in_paths",
+    "-Dunused_lifetimes",
+    "-Drust_2021_prelude_collisions"
```

Comment:
Because these are `rustc` lints, it might be nice to have them warn by default even when not using clips. Especially the `elided_lifetimes_in_paths` - I'm pretty sure I'll trip up on this a lot!